utils/build.ps1

```diff
@@ -774,7 +774,7 @@ function Get-PinnedToolchainVersion() {
   if (Test-Path variable:PinnedVersion) {
     return $PinnedVersion
   }
-  return "5.10.1"
+  throw "PinnedVersion must be set"
```

Comment:
I don't get the body of this function - the `variable:PinnedVersion` expansion is being tested as a path, but not relative to any location, so how does this work?

Reply:
I think `Test-Path variable:FOO` is an idiom to check if variable `FOO` is set.